### ISSUE TITLE
shiny refresh page not working anymore

### ISSUE THREAD

**Issue by cmber** [ficonsulting/RInno#117]
Hi all;
First of all thanks for such a great application. It's been very useful.
I'm working with a standalone shiny application that I created with the latest version of RInno using Chrome as the default browser. In order to refresh one of the tabs I have used:

UI:
shinyjs::useShinyjs(),
shinyjs::extendShinyjs(text = "shinyjs.refresh = function() { location.reload(); }")

and in the server side:
observeEvent(input$reset, {shinyjs::js$refresh()})

Everything was working fine until a couple of days ago. For some reason now when I click on the reset button the browser is not refreshed and if I click again I get an error saying that the site (page) can't be reached. The funny thing is that when I run the app in RStudio (either using the shiny window or the web browser) it work perfectly. I tried Firefox as well and get the same problem.

Any ideas?

Best,
Christian

**Comment by cmber** [comment]
I've tested different alternatives for resetting the pages of my app. All of them work when the ui.R is run from RStudio or R alone. I've uninstalled and reinstalled R, Inno, RInno.... but when the app is packaged and installed later, the reset command (from shinyjs or using html tag) does not work. It would be good is someone can give me a hint of what could be happening and if there is any solution. Thanks!
